### ISSUE TITLE
Article Editing Permissions shouldn't be restricted to Admins only

### ISSUE THREAD

**Issue by TurtleWolfe** [jfox015/Bonfire-News#25]
I've tried opening up the permissions so that users can edit their own entries, but if I filter for just their articles, it seems like I'd be doing the same thing to the admin... so in theory the admin or an editor should be able to edit any article and the users should only be able to edit theirs. Am I overlooking an easy permission somewhere. I don't mind if they see them... I'm running into similar issues with who is allowed to delete an article, they seem to go into limbo, I still have them in the database, but they disappear from the live site. Most recent live version has been moved to http://brolu.net/, I'm starting to get feedback from beta testers now, so hold onto your hat.

**Comment by jfox015** [comment]
@TurtleWolf  Bonfire uses the concept of "soft deletes" so that you can delete articles from the main site, but they're simply tagged as deleted in the DB< not actually removed. It's kind of like the Windows Recycle Bin. You can permanently delete them or recover them.

Permissions is a bit tricky when it comes to who should edit what. I agree that Admin and Editors should have blanket permissions to edit anything, but users should be able to edit their own articles as well. This shouldn't be a terribly difficult problem to solve, just an extra layer of checks in the queries that the different news queries perform.

**Comment by TurtleWolfe** [comment]
Yes, I understand soft deletes... but along with tabs for draft, archived, published and review, there is a deleted tab... if someone that doesn't have permission, tries to delete, they just disappear... still in the database but not in the "soft" deletes tab. I'm also not getting any response under sorting for author or category.

**Comment by jfox015** [comment]
@TurtleWolf Can you log both of those as separate bugs? Thanks.
